
Error:
Traceback (most recent call last):
  File "C:\AVRL\online-judge-v\online_judge\codes\52d85dab-4a78-4bcd-a492-724d0f989bc3_code.py", line 1, in <module>
    a = int(input())
EOFError: EOF when reading a line
